app/build.gradle

```diff
@@ -20,7 +20,7 @@ android {
 
     buildTypes {
         release {
-            minifyEnabled true
+            minifyEnabled false
```

Comment:
Please keep in mind that minifyEnabled is very useful for decreasing the size of the application and it does assist on preventing reverse engineer of the app code for exploits. If you need this to stack trace some crash in production environment, maybe leave a TODO to change later.